include/wx/valtext.h

```diff
@@ -68,31 +74,75 @@ class WXDLLIMPEXP_CORE wxTextValidator: public wxValidator
 
     wxTextEntry *GetTextEntry();
 
+    // strings & chars inclusions:
+    // ---------------------------
+
     void SetCharIncludes(const wxString& chars);
+    void AddCharIncludes(const wxString& chars);
+
     void SetIncludes(const wxArrayString& includes) { m_includes = includes; }
-    inline wxArrayString& GetIncludes() { return m_includes; }
+    void AddInclude(const wxString& include)
+    {
+        if ( !include.empty() )
+            m_includes.Add(include);
```

Comment:
Please prefer using std-like methods in the new code, i.e. `push_back()` instead of `Add()`.

Reply:
done.